projects/angular/src/layout/nav/_header.clarity.scss

```diff
@@ -155,6 +155,7 @@
       align-items: center;
       height: header-variables.$clr-header-height;
       text-decoration: none;
+      gap: 0;
```

Comment:
As discussed offline, these `gap: 0` probably wouldn't be needed when we revert the change for the anchor.

Reply:
corrected in commit bf22d26